usecases/modules/modules.go

```diff
@@ -1142,7 +1141,9 @@ func (p *Provider) EnabledBackupBackends() []modulecapabilities.BackupBackend {
 }
 
 func (p *Provider) UsageEnabled() bool {
-	if module := p.GetByName(modusagegcs.Name); module != nil {
+	// TODO: used as string here to avoid circular dependency
+	// need to find a better way to do this may be GetAll instead of GetByName
+	if module := p.GetByName("usage-gcs"); module != nil {
```

Comment:
can you just relay on this check `if module.Type() == modulecapabilities.Usage`? or do you have to pass module's name here? if so, then add a method `IsUsageModule(name string)` and pass that name in the place where you are using this method

Reply:
the problem we have circular dependency that what makes this limiation it will require another refactor :(